app/helpers/users_helper.rb

```diff
@@ -1,7 +1,9 @@
 module UsersHelper
   def gravatar_for user
-    gravatar_id = Digest::MD5::hexdigest user.email.downcase
-    gravatar_url = "#{Settings.gamer_community.links.gravatar}#{gravatar_id}"
-    image_tag gravatar_url, alt: user.name, class: "gravatar"
+    if user.avatar
+      user.avatar.url
+    else
+      "default.png"
+    end
```

Comment:
```suggestion
    user.avatar ? user.avatar.url : "default.png"
```
and move default.png to settings file and call it here

Reply:
done